satellite/discovery/service.go

```diff
@@ -120,7 +105,7 @@ func (discovery *Discovery) refresh(ctx context.Context) (err error) {
 
 			limiter.Go(ctx, func() {
 				// NB: FetchInfo updates node uptime already
```

Comment:
i think this comment is wrong. FetchInfo seems to simply fetch info, but not update node uptime.

Reply:
ah yes, that was an old note from when FetchInfo was in kad + transport observers.